src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -350,15 +373,20 @@ - (void)updateVisiblePagesShouldNotifyDelegate:(BOOL)shouldNotifyDelegate {
 
     [self didChangeCenterPageIndexFrom:oldCenterPageIndex to:_centerPageIndex];
 
-    // Prioritize displaying the currently visible page.
-    if (![self isDisplayingPageForIndex:_centerPageIndex]) {
-      [self displayPageAtIndex:_centerPageIndex];
+    if (_pageInset != 0) {
+      // When multiple images are visible load them all.
```

Comment:
This comment seems inaccurate and doesn't reflect the actual condition it's based upon. Maybe it should read "Load all visible insetted pages immediately."?